### DISCUSSION TITLE
Somatic Hypermutation Status of IGHV status

### DISCUSSION THREAD

**Discussion by xmy1990** [milaboratory/mixcr#1678] [Q&A] [answered]
Hi,

command,
mixcr exportClones -vIdentityPercents -vBestIdentityPercent -dIdentityPercents -dBestIdentityPercent -jIdentityPercents -jBestIdentityPercent test.contigs.clns clones.tsv

clones_IGH.tsv,
cloneId readCount       readFraction    targetSequences targetQualities allVHitsWithScore       allDHitsWithScore       allJHitsWithScore    allCHitsWithScore       allVAlignments  allDAlignments  allJAlignments  allCAlignments  nSeqFR1 minQualFR1      nSeqCDR1    minQualCDR1      nSeqFR2 minQualFR2      nSeqCDR2        minQualCDR2     nSeqFR3 minQualFR3      nSeqCDR3        minQualCDR3     nSeqFR4      minQualFR4      aaSeqFR1        aaSeqCDR1       aaSeqFR2        aaSeqCDR2       aaSeqFR3        aaSeqCDR3       aaSeqFR4    refPoints        vIdentityPercents       vBestIdentityPercent    dIdentityPercents       dBestIdentityPercent    jIdentityPercents   jBestIdentityPercent

which of the "vIdentityPercents       vBestIdentityPercent    dIdentityPercents       dBestIdentityPercent    jIdentityPercents   jBestIdentityPercent" can use to evaluate  Somatic Hypermutation Status of IGHV status?

Thanks

**Comment by xmy1990** [comment]
IGHV status, Somatic Hypermutation Status of immunoglobulin heavy variable (IGHV) genes

**Comment by mizraelson** [comment, marked as answer]
Hi,

I think the best field is `-nMutationsRate VRegionTrimmed`. This will give you the number of mutations in the V gene divided by its length. It is recommended to first run [`mixcr findalleles`](https://mixcr.com/mixcr/reference/mixcr-findAlleles/) and then exportClones, as in that case you will have less of a chance of confusing mutations (SHMs) with allelic variants.
